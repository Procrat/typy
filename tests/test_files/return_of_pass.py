def f():
    pass

f().non_attribute
