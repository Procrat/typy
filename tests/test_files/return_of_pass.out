None has no attribute non_attribute
